### PR TITLE
Simple examples how to use pREST

### PR DESCRIPTION
**Pull request by crgimenes**
Simple examples of how to use pREST in your Go code

### CODE REVIEW

File: custom-middlewares/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+export PREST_JWT_DEFAULT=false
+```
+
+Start example with following command:
+
+```console
+go run main.go
+```
+
+Then you can access the url http://localhost:3000/ping and in this case the sertver will return `Calling custom middleware` and then `Pong!` as response.
```
Comment (marioidival) [inline custom-middlewares/README.md]: typo, sertver

File: extending-framework/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Run Extending (framework)
+
+To run extending framework example with local database configure the following envaironment variables
```
Comment (marioidival) [inline extending-framework/README.md]: typo envaironment

**Review by marioidival — CHANGES_REQUESTED** [review]
some typos

**Review by marioidival — APPROVED** [review]
👑  LGTM 👑

File: custom-middlewares/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Run Custom Middlewares
+
+To run extending framework example with local database configure the following envaironment variables
```
Comment (lerrua) [inline custom-middlewares/README.md]: typo envaironment (environment)

File: extending-framework/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+In this example we desabled the JWT with the `PREST_JWT_DEFAULT=false` environment variable.
```
Comment (lerrua) [inline extending-framework/README.md]: typo desabled (disabled)

File: custom-middlewares/README.md (hunk 3 above)
Comment (crgimenes) [reply custom-middlewares/README.md]: @lerrua typo corrected :D

File: extending-framework/README.md (hunk 4 above)
Comment (crgimenes) [reply extending-framework/README.md]: @lerrua typo corrected :D

**Pull request merged**